packages/api/apollo-server/queries/index.ts

```diff
@@ -0,0 +1,7 @@
+import users from './users'
+
+export default {
+  Query: {
+    users
```

Comment:
It's not clear what this query does? Does it query all users? single user?

Reply:
> It's not clear what this query does? Does it query all users? single user?

I was thinking about to create another PR to define all the resolvers. Don't want to make a PR too big.